src/components/DatePicker.vue

```diff
@@ -77,7 +77,6 @@
                 :belongsToThisMonth='day.belongsToThisMonth'
                 :checkIn='checkIn'
                 :checkOut='checkOut'
-                :currentDateStyle='currentDateStyle'
```

Comment:
@ayaz323 please update your branch not to delete the last changes that were implemented 🙏🏼